src/cluster.c

```diff
@@ -237,7 +237,7 @@ void restoreCommand(client *c) {
 
     rioInitWithBuffer(&payload,c->argv[3]->ptr);
     if (((type = rdbLoadObjectType(&payload)) == -1) ||
-        ((obj = rdbLoadObject(type,&payload,key->ptr,c->db->id,NULL)) == NULL))
+        ((obj = rdbLoadObject(type,&payload,key->ptr,c->db,c->db->id,NULL)) == NULL))
```

Comment:
Right, only that db might be NULL (in RDB verification scenario).

Reply:
Fixed by removing dbid, as it was only used for module loading and in that case, the code using it would return before using it if it was in rdb check mode. Added a validation db is not NULL before using it.
However, there was some accommodation to do in order to adjust to the option of NULL db when loading HFE, since db is used to connect all fields with expiry.